circle.yml

```diff
@@ -145,74 +144,53 @@ commands:
       packageName:
         description: Name of the package to restore, for example "server" or "desktop-gui"
         type: string
-      packagePath:
-        description: Path to package.json file
-        type: string
     steps:
       - restore_cache:
           name: Restoring cache for << parameters.packageName >>
-          key: v{{ .Environment.CACHE_VERSION }}-{{ arch }}-{{ .Branch }}-deps-<< parameters.packageName >>-{{ checksum "<< parameters.packagePath >>" }}
+          key: v{{ .Environment.CACHE_VERSION }}-{{ arch }}-{{ .Branch }}-deps-<< parameters.packageName >>-{{ checksum "yarn.lock" }}
 
   restore-caches:
     description: need to restore a separate cache for each package.json
     steps:
       - restore-cache:
           packageName: cli
-          packagePath: cli/package.json
       - restore-cache:
           packageName: root
-          packagePath: package.json
       - restore-cache:
           packageName: coffee
-          packagePath: packages/coffee/package.json
       - restore-cache:
           packageName: desktop-gui
-          packagePath: packages/desktop-gui/package.json
       - restore-cache:
           packageName: driver
-          packagePath: packages/driver/package.json
       - restore-cache:
           packageName: example
-          packagePath: packages/example/package.json
       - restore-cache:
           packageName: electron
-          packagePath: packages/electron/package.json
       - restore-cache:
           packageName: extension
-          packagePath: packages/extension/package.json
       - restore-cache:
           packageName: https-proxy
-          packagePath: packages/https-proxy/package.json
       - restore-cache:
```

Comment:
why do we still need to cache each package? We shouldn't be caching node_modules directly at all right? only the .cache folder

Reply:
In hindsight, I think you are correct. We should not cache each packages' node_modules as a change to the yarn.lock file will require a yarn which will update across all packages anyway. I'll make the required changes to this.